internal/cli/alpha/manifestgen/testdata/cap.type.test.empty.yaml

```diff
@@ -0,0 +1,21 @@
+ocfVersion: 0.0.1
+revision: 0.2.0
+kind: Type
+metadata:
+  prefix: "cap.type.test"
+  name: empty
+  displayName: Type test.empty
+  description: Description of the Type
+  documentationURL: https://example.com
+  supportURL: https://example.com
+  maintainers:
+    - email: dev@example.com
+      name: Example Dev
+      url: https://example.com
+spec:
+  jsonSchema:
+    # TODO(ContentDeveloper): Adjust the JSON schema if needed.
+    value: |-
+      {
+        "type": "object"
+      }
```

Comment:
Why the schema looks like this? In the first/second review iteration you unified all JSON schema as requested, and now it's still different depending on some conditions I'm not aware of 🤔  Unfortunately it is a regression between the previous iteration I tested.